File: dcli/template/java/src/main/java/success/Target.java
package success;

import java.util.*;

public class {{ targetFile }} {

    public static void main() {

    }

    public static {% if targetGenerics != "" %}<{{ targetGenerics }}> {% endif %}{% if targetMethodReturn == "" %}void {% else %}{{ targetMethodReturn }}{% endif %} {{ targetMethod }}({{ targetMethodArgs }}) {
        //TODO write your own solution
        return {% if targetDefaultReturn == None %} null {% else %}{{ targetDefaultReturn }}{% endif %};
    }
}

File: dcli/template/kotlin/src/main/kotlin/success/Target.kt
package success

import java.util.*

class {{ targetFile }} {
    companion object {
        fun {{ targetMethod }}({{ targetMethodArgs }}): {% if targetMethodReturn == "" %}Unit{% else %}{{ targetMethodReturn }}{% endif %} {
            //TODO write your own solution
            {% if targetMethodReturn != "" %}return {{ targetMethodReturnValue }};{% endif %}
        }
    }
}
